.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Build and push container
+
+on:
+  push:
+    branches:
```

Comment:
The branch name should be here, just like on the 1.8.6 branch. Otherwise it's good to go.

Reply:
sure thing, it's back

Reply:
omg, I put `1.8.6` there :joy: 
thanks for fixing that up :+1: